systems/ukama-agent/docker-compose.yml

```diff
@@ -112,14 +112,14 @@ services:
    - DB_USER=postgres
    - DB_PASSWORD=Pass2020!
    - DB_HOST=postgresd-ukama-agent
-   - QUEUE_URI=amqp://guest:guest@$${LOCAL_HOST_IP}:5672
+   - QUEUE_URI=amqp://guest:guest@${LOCAL_HOST_IP}:5672
    - SYSTEM=ukamaagent
    - SHOVEL_SRCEXCHANGEKEY=event.cloud.*.*.*
-   - SHOVEL_SRCURI=amqp://guest:guest@$10.1.0.11:5672
+   - SHOVEL_SRCURI=amqp://guest:guest@10.1.0.11:5672
```

Comment:
Shouldn't this IP in shovel url also need to be `LOCAL_HOST_IP`

Reply:
This is what I also think, but I have no idea why Vishal chooses to keep the internal docker container IP, so I leave it that way until I am sure it is safe to change.